src/main/java/hudson/remoting/Launcher.java

```diff
@@ -244,6 +268,90 @@ public void run() throws Exception {
         System.exit(0);
     }
 
+    private SSLSocketFactory getSSLSocketFactory()
```

Comment:
Seems SSLSocketFactory nullability implicitly spreads widely outside this method. Maybe safe, but annotations definitely do not hurt ;)